deploy/vagrant_hosts

```diff
@@ -2,32 +2,32 @@
 
 ; Shared Component
 [admin_terminal]
-admin_terminal ansible_ssh_host=10.0.1.2
+admin_terminal_0 ansible_ssh_host=10.0.1.2
```

Comment:
:+1: but perhaps we should start with `_1` to be consistent with the other entries (elasticsearch, file_server)

Reply:
This should be an easy fix: it's very rare that anything refers to a specific host; almost all references are by group.

Reply:
Narrator: it actually **was** an easy fix.